src/controllers/EvidenceUploadController.ts

```diff
@@ -34,32 +39,98 @@ export class EvidenceUploadController extends BaseController<OtherReason> {
         return appeal.reasons?.other;
     }
 
+    private async renderUploadError(appeal: Appeal, text: string): Promise<void> {
+        const that = this;
```

Comment:
I doubt `that = this` technique is needed here. Cause it is private method on the controller `this` should have correct value.